### ISSUE TITLE
Lint/SafeNavigationChain should allow `in?` (ActiveSupport) method

### ISSUE THREAD

**Issue by johnnyshields** [rubocop-hq/rubocop#8500]
If ActiveSupport is required, the `.in?` method should be permitted in a safe-navigation chain:

```ruby
foo&.to_sym.in?([:baz, :qux])
```

Currently this raises the error: `Lint/SafeNavigationChain: Do not chain ordinary method call after safe navigation operator.`

RuboCop version: 0.89.0

**Comment by johnnyshields** [comment]
Thank you!
